tracing/src/lib.rs

```diff
@@ -1001,6 +1001,7 @@ pub mod __macro_support {
     use crate::{collect::Interest, Metadata};
     use core::fmt;
     use core::sync::atomic::{AtomicU8, Ordering};
+    pub use core::{concat, format_args, iter::Iterator, option::Option};
```

Comment:
can we add a comment here documenting why we re-export these? the primary reason I wanted a regression test was to guard against making changes that re-break `tracing` inside of crates named `core`, so it would be nice to at least include a comment here explaining the re-export and why we use it. that way, anyone who might otherwise think that this should be removed in favor of just referencing the fully qualified paths from libcore will understand why we don't do that.

Reply:
Thanks for the review! Added the explanation—let me know how it looks!